src/App.tsx

```diff
@@ -2,108 +2,22 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { Post } from './types/Post';
+import { PostList } from './components/PostList';
+
+const fullPosts: Post[] = postsFromServer.map(post => ({
```

Comment:
I think it would be better
```suggestion
const preparedPosts: Post[] = postsFromServer.map(post => ({
```